.github/workflows/create-design-review-issue.js

```diff
@@ -70,6 +70,8 @@ When attending a design review please read through the designs _before_ the desi
 We rotate our design review between times that work better for different parts of the world as described in our [design review documentation](https://github.com/ampproject/amphtml/blob/main/docs/design-reviews.md), but you are welcome to attend any design review. If you cannot make any of the design reviews but have a design to discuss please let mrjoro@ know on [Slack](https://github.com/ampproject/amphtml/blob/main/docs/contributing.md#discussion-channels) and we will find a time that works for you.
```

Comment:
Comment for line 55: Should we switch from bit.ly to the proper URL https://meet.google.com/umi-owgo-wen?

Reply:
I believe we can modify the `bit.ly` URL any time, and since it's visible, it's best if we keep that updated separately.

Reply:
SGTM. An alternative is to create a [go.amp.dev short-link](https://github.com/ampproject/amp.dev/blob/future/platform/config/go-links.yaml), but that shouldn't block this PR.